packages/kbn-test/src/functional_tests/cli/start_servers_cli.js

```diff
@@ -0,0 +1,43 @@
+import getopts from 'getopts';
+import { startServers } from '../../';
+import { FUNCTIONAL_CONFIG_PATH } from '../../functional_tests/lib';
+
+const defaultConfigPath = FUNCTIONAL_CONFIG_PATH;
+
+/**
+ * Start servers
+ * @param {configPath}     path to config
+ */
+export function startServersCli() {
+  const {
+    config,
+    help,
+  } = processArgv();
+
+  if (help) return displayHelp();
+
+  startServers(config);
+}
+
+
+function processArgv() {
+  const options = getopts(process.argv.slice(2)) || {};
+
+  return {
+    config: options.config || defaultConfigPath,
+    help: options.help,
+    rest: options._,
+  };
+}
+
+
+function displayHelp() {
+  console.log();
```

Comment:
I personally prefer using `dedent` and a long template string in situations like this. I think it's easier to maintain:

```js
log.write(dedent`
  Functional Tests' Servers

  Usage:  node scripts/functional_tests_server [options]

  --config      Option to pass in a config
  --help        Displays this menu and exits
`);
```